Content.Server/Cargo/Components/StationCargoBountyDatabaseComponent.cs

```diff
@@ -32,4 +33,16 @@ public sealed partial class StationCargoBountyDatabaseComponent : Component
     /// </summary>
     [DataField]
     public HashSet<string> CheckedBounties = new();
+
+    /// <summary>
+    /// The time at which players will be able to skip the next bounty.
+    /// </summary>
+    [DataField("nextSkipTime", customTypeSerializer: typeof(TimeOffsetSerializer))]
+    public TimeSpan NextSkipTime = TimeSpan.Zero;
+
+    /// <summary>
+    /// The time between skipping bounties.
+    /// </summary>
+    [DataField("skipDelay")]
```

Comment:
```suggestion
    [DataField]
```

Reply:
ConsoleComponent has an argument for DataField, but DatabaseComponent should not? Is there a rule for when I should and shouldn't do that?

Reply:
the tag name is now autogenerated from the name
so SkipDelay first letter lowercase is skipDelay, so you dont need it

Reply:
ahh got it, thanks